docs/docs/adrs/adr-013-equivocation-slashing.md

```diff
@@ -0,0 +1,169 @@
+
+**Power:** We pass the sum of the voting power of the misbehaving validator and the power of all the ongoing undelegations and redelegations. This is a slightly more aggressive approach than just providing the voting `power` at the evidence height. If we assume that the `slashFactor` is 5%, then the `power` we pass is `validatorPower + totalPower(undelegations) + totalPower(redelegations)`.
+Hence, when the `Slash` method slashes all the undelegations and redelegations it would end up with `0.05 * validatorPower + 0.05 * totalPower(undelegations) + 0.05 * totalPower(redelegations) - 0.05 * totalPower(undelegations) - 0.05 * totalPower(redelegations) = 0.05 * validatorPower` and hence it would slash 5% of the `validatorPower` at evidence height.
+
+**Storing evidence:** As mentioned, we do not expire evidence with this aggressive approach. However, because we do not want to slash the same validator more than once for the same infraction, we have to store the evidence in some _cache_ in order to check if we have already slashed that validator. We intend to do the following:
+```go
+// hashEvidence should only use the signed parts of the evidence to generate the hash
+// therefore, it should only use VoteA and VoteB to avoid someone changing other parts of DuplicateVoteEvidence
+// and resubmitting
+key := hashEvidence(dve DuplicateVoteEvidence) 
+
+if key not in cache {
+	store (key, validatorAddr) in cache
+	perform slashing
+} else {
+	// do not slash
+}
+```
+To prevent this cache from growing arbitrarily big, we introduce additional checks. If the evidence  is for a validator that is not in the active validator set we skip from storing it in the cache, etc.
+We also periodically clean up the cache by checking if the validator `validatorAddr` of a cache entry  is still in the active set and if not we remove that entry from the cache.
+
+### Positive
+With the proposed approach we can quickly implement slashing functionality on the provider chain for consumer chain equivocations.
+This approach does not need to change the staking module and therefore does not change in any way how slashing is performed today for a single chain.
+
+### Negative
+We _definitely_ slash more when it comes to undelegations and redelegations because we slash for all of them without considering an `infractionHeight`.
+We _potentially_ slash more than what we would have slashed if we knew the voting `power` at the corresponding `infractionHeight` in the provider chain.
```

Comment:
```suggestion

- We _definitely_ slash more when it comes to undelegations and redelegations because we slash for all of them without considering an `infractionHeight`.
- We _potentially_ slash more than what we would have slashed if we knew the voting `power` at the corresponding `infractionHeight` in the provider chain.
```

Reply:
Also add a bullet point about the evidence age.